roles/ces_common/tasks/configure.yml

```diff
@@ -157,7 +155,7 @@
 - name: configure| Check CES enabled on all nodes
   assert:
     that:
-      - "item in scale_ces_enable_status.stdout"
+      - hostvars[item]['scale_daemon_nodename'] in scale_ces_enable_status.stdout
```

Comment:
@acch Can we keep inventory_hostname instead of scale_daemon_nodename ?

Reply:
scale_ces_enable_status is produced by `mmces node list`, which will print the nodename rather than the inventory name.

Reply:
@acch Now i got it. Thanks